Makefile

```diff
@@ -2,8 +2,7 @@
 
 #GOOS=linux
 
-GO=CGO_ENABLED=0 GO111MODULE=on go
-GOCGO=CGO_ENABLED=1 GO111MODULE=on go
+GO=CGO_ENABLED=0 go
```

Comment:
This was removed in reference PR.
CGO_ENABLED=0  only need on the build so no reason for the variable.